src/components/EditPageAndCard/EditSpot/EditSpotCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { useParams } from 'react-router-dom';
+import { EntityCard } from 'components/CreateSpotAndPersona/CreateCard/EntityCard';
+import { CardType } from '../../../global/graphqls/schema';
+import { useMutation, useQuery } from '@apollo/react-hooks';
```

Comment:
I think "global/graphqls/schema" is just enough

Reply:
change